push.sh

```diff
@@ -2,14 +2,18 @@
 
 function push {
     if [ "$1" = "-h" ]; then
-    echo " Sends a request to the pushover.net server"
-    echo "Takes two positionnal arguments the title and the body of the push notification"
+        echo " Sends a request to the pushover.net server"
+        echo "Takes two positionnal arguments the title and the body of the push notification"
         return  
     fi
 
-    #requires COSTUMISATION
-    curl -s -F "token=TOKENNUMBER" \
-    -F "user=USERID" \
+    CDIR="${XDG_CONFIG_HOME:-$HOME/.config}/push"
+    mkdir -p "$CDIR"
+    [ ! -f "$CDIR/token" -o ! -f "$CDIR/user_id" ] && \
+        { echo "Auth tokens must be supplied."; return 1; }
```

Comment:
Should there be a mechanism to write the files?
Should there be the possibility to change these values?